src/kv/committable_tx.h

```diff
@@ -398,12 +422,19 @@ namespace kv
       std::string commit_evidence;
 
       // This is a signature and, if the ledger chunking flag is enabled, we
-      // want the host to create a chunk when it sees this entry.
+      // want the host to create a chunk when it sees this entry. On backups, we
+      // also trigger a ledger chunk when the primary creates a snapshot (which
+      // also triggers a new ledger chunk).
```

Comment:
I think the comment on backups applies to the primary node too. In fact, it's probably to leave the details about primary/backup nodes out of the KV store implementation.